TourGuide_Horde/37_Alterac_Mountains.lua

```diff
@@ -3,12 +3,14 @@ return [[
 F Tarren Mill
 A Prison Break In |N|From Magus Wordeen Voidglare (61.6, 20.9)| |Z|Hillsbrad Foothills| |NODEBUG| |QID|544| |NODEBUG|
 A Stone Tokens |N|From Keeper Bel'varil| |QID|556|
+K Mountain Lion |L|5810|
+C Helcular's Revenge (Part 2) |QID|553| |QO|Flame of Uzel charged|
```

Comment:
Remove the |QO| tag here since C is for completion and getting the last flame will have completed the quest. Instead, you can make this a note (with a coord) for Flame of Uzel.

Reply:
Will change.